explorer/explorerroutes.go

```diff
@@ -272,16 +279,18 @@ func (exp *explorerUI) Ticketpool(w http.ResponseWriter, r *http.Request) {
 	exp.MempoolData.RUnlock()
 
 	str, err := exp.templates.execTemplateToString("ticketpool", struct {
-		Version     string
-		NetName     string
-		ChartData   []*dbtypes.PoolTicketsData
-		GroupedData *dbtypes.PoolTicketsData
-		Mempool     *dbtypes.PoolTicketsData
+		Version      string
+		NetName      string
+		ChartsHeight uint64
```

Comment:
This will be used to indicate the latest block fetched on the frontend?

Reply:
The block height corresponding to the charts data.  Unused, but can be